javascript/javascript2/week1/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="main.css" /><link
+    <title>hyfBayHelpers task</title>
+</head>
+<body>
+    <script src="hyfBayHelpers.js"></script>
+    <script src="main.js"></script>
+    
```

Comment:
Hi Violeta. Good work!
I have a few comments. 
Usually we use empty lines in JS files, it gives us more readable code. In HTML files we don’t do it.

Reply:
I just wanted to comment on this to clarify it a little bit. In general adding blank lines in Js, HTML, or another file is fine if it improves the readability. you can see the style guideline of HTML here for reference: https://www.w3schools.com/html/html5_syntax.asp
But I can see that here a lot of blank lines do not improve the readability, but it's not a general rule in HTML to never add blank lines.